scripts/docker.mk

```
@@ -0,0 +1,14 @@
+# Default configuration
+IMG_NAME := $(DOCKER_PUSH_REPOSITORY)$(DOCKER_PUSH_DIRECTORY)/$(APP_NAME)
+TAG := $(DOCKER_TAG)
+
+.PHONY: release
+
+release: build-image push-image
```

Let's have two colons here, so one can extend it on component side.
```suggestion
release:: build-image push-image
```